esphome/components/logger/logger.h

```diff
@@ -277,7 +281,7 @@ class Logger : public Component {
 #endif
 
 #if defined(USE_ESP32) || defined(USE_LIBRETINY) || defined(USE_ZEPHYR)
-  const char *HOT get_thread_name_() {
+  const char *HOT get_thread_name_(char *buff) {
```

Comment:
```suggestion
  const char *HOT get_thread_name_(
  #ifdef USE_ZEPHYR
      char *buff
  #endif
    ) {
```

or split for better readability

Reply:
done